demos/pedestrian_tracker_demo/cpp/src/tracker.cpp

```diff
@@ -35,7 +35,7 @@ DetectionLog ConvertTracksToDetectionLog(const ObjectTracks& tracks) {
     DetectionLog log;
 
     // Combine detected objects by respective frame indices.
-    std::map<int, TrackedObjects> objects;
+    std::map<int64_t, TrackedObjects> objects;
```

Comment:
why data type changed? What is the reason?

Reply:
@vladimir-dudnik the model API uses the int_64_t type for "frameId", then it is the first key in this structure

Reply:
thanks, ok, then